### PR TITLE
Some minor fixes and enhancements.

### PR DESCRIPTION
**Pull request by WitherredAway**
- Add missing start check on some commands.
- Non-admin users can now check the prefix.
- Prevent users from requesting trades and battles with suspended users.
- Users suspended mid-trade can now cancel the trade.

### CODE REVIEW

**Comment by WitherredAway** [comment]
Note: All this is untested, so please double check! Thanks! :wiggleflushed:

File: cogs/auctions.py [hunk 1]
```diff
@@ -162,7 +162,7 @@ async def auction(self, ctx):
         await ctx.send_help(ctx.command)
 
     @commands.has_permissions(administrator=True)
-    @auction.command()
+    @auction.command(case_insensitive=True)
```
Comment (pikaninja) [inline cogs/auctions.py:165]: case_insensitive *does not go here*
case_insensitive is passed into *group* commands or the main bot class, its not set per command

Comment (pikaninja) [reply cogs/auctions.py:165]: Same goes for all the other locations you added case_insensitive

Comment (WitherredAway) [reply cogs/auctions.py:165]: wait, oh, my bad

Comment (WitherredAway) [reply cogs/auctions.py]: Fixed, thank you

**Comment by WitherredAway** [comment]
Going to open this PR again from a different branch

**Pull request closed without merging**